README.md

```diff
@@ -110,11 +110,12 @@ wget https://raw.githubusercontent.com/open-rmf/rmf/main/rmf.repos
 vcs import src < rmf.repos
 ```
 
-Install dependencies via `rosdep`.
+Update your rosdep definitions and install dependencies via `rosdep`.
 Replace `humble` with ROS 2 distro of your choice.
 
 ```bash
 cd ~/rmf_ws
+rosdep update
 rosdep install --from-paths src --ignore-src --rosdistro humble -y
```

Comment:
```suggestion
# source your preferred ROS distribution. Eg: `source /opt/ros/rolling/setup.bash`
rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
```

Reply:
And perhaps delete the `source /opt/ros/humble/setup.bash # replace humble with ROS 2 distro of choice.` step below.

Reply:
@arjo129 could you address the second comment here too?